docs/aquacomputer_d5next.rst

```diff
@@ -56,6 +57,9 @@ The High Flow Next exposes +5V voltages, water quality, conductivity and flow re
 A temperature sensor can be connected to it, in which case it provides its reading
 and an estimation of the dissipated/absorbed power in the liquid cooling loop.
 
+The Aquastream XT pump exposes temperature readings for the coolant, external sensor
+and fan IC. It also exposes pump and fan voltages, as well as pump current.
```

Comment:
Pump and fan RPM are also supported.

Reply:
Thanks, fixed this and the other comment.